frontend/src/Login/Login.tsx

```diff
@@ -1,15 +1,26 @@
+import { Box } from '@mui/material';
 import { useTranslation } from 'react-i18next';
-
+import './Login.css';
 
 function Login() {
-  const {t, i18n} = useTranslation();
+  const {t} = useTranslation();
   return (
-      <div>
-        <p>{t('title')}</p>
-        <form onSubmit={(e) => {i18n.changeLanguage(i18n.language === 'en' ? 'ru' : 'en'); e.preventDefault()}}>
+      <Box display="flex" flexDirection="column" alignItems="center">
+        <h2>{t('login.title')}</h2>
+        <Box component="form" display="flex" flexDirection="column" alignItems="center">
+          <label className="login-form-label">
+            <span className="label-wrapper">{t('login.login')}:</span>
+            <input type="text"/>
+          </label>
+          <br/>
+          <label className="login-form-label">
+            <span className="label-wrapper">{t('login.password')}:</span> 
+            <input type="password"/>
+          </label>
+          <br/>
```

Comment:
It would be nice to make smth like functions "wrong password message" and "wrong login message" (and add these messages to translations.json) 
But it can be completed later

Reply:
Also added :+1: